tests/integration/targets/manage_vcenter_host_connection_test/run.yml

```diff
@@ -0,0 +1,17 @@
+- hosts: localhost
+  gather_facts: no
+  collections:
+    - community.general
+
+  tasks:
+    - name: Vcsim
+      ansible.builtin.import_role:
+        name: prepare_soap
+
+    - name: ESXi
+      ansible.builtin.import_role:
+        name: prepare_esx
```

Comment:
I see you here run both `prepare_soap` and `prepare_esx`. Sould just one prepare_soap be enough? could we maybe rename the `prepare_soap` to `prepare_vcsim` and add a paramter, to change the `command` ? so we parameterize it as we need?